test/unittests/instructions_test.cpp

```diff
@@ -138,6 +150,11 @@ TEST(instructions, compare_with_evmc_instruction_names)
     const auto* evmc_tbl = evmc_get_instruction_names_table(EVMC_MAX_REVISION);
     for (size_t i = 0; i < instr::traits.size(); ++i)
     {
+        // Skip DUPN and SWAPN for Cancun. They are not defined in evmc
+        // TODO: Define DUPN and SWAPN in evmc
+        if (instr::traits[i].since == EVMC_CANCUN &&
```

Comment:
Nit: checking for Cancun is redundant here, DUPN/SWAPN are only since Cancun always.